content/utilities/PlanetOverview/plugin.js

```diff
@@ -0,0 +1,301 @@
+    updateButton.innerText = "Update";
+    updateButton.style.marginRight = "10px";
+    updateButton.addEventListener("click", () => {
+      try {
+        dynamicLabel.innerText = `Top ${this.topX} of PlanetType: ${this.planetType} up Lvl: ${this.minLevel}`;
+        this.renderPlanets();
+      } catch (err) {
+        console.error("Unable to update", err);
+      }
+    });
+
+    // Button "Un.Trans"
+    const unconfirmedButton = document.createElement("button");
+    unconfirmedButton.innerText = "Un.Trans";
+    unconfirmedButton.style.marginRight = "10px";
+    unconfirmedButton.addEventListener("click", () => {
+      updateTxStatus();
+      unconfirmed();
+      updateTx();
+      this.renderPlanets();
+    });
+
+    // Slider for count of total list lines
+    const topSlider = document.createElement("input");
+    topSlider.style.marginTop = "13px";
+    topSlider.style.width = "37%";
+    topSlider.type = "range";
+    topSlider.value = `${this.topX}`;
+    topSlider.min = 1;
+    topSlider.max = 100;
+    topSlider.onchange = (evt) => {
+      try {
+        this.topX = parseInt(evt.target.value, 10);
+        dynamicLabel.innerText = `Top ${this.topX} of PlanetType: ${this.planetType} up Lvl: ${this.minLevel}`;
+        this.renderPlanets();
+      } catch (err) {
+        console.error("Unable to parse number", err);
+      }
+    };
+
+    // label for dynamic information
+    const dynamicLabel = document.createElement("label");
+    dynamicLabel.style.width = "50%";
+    dynamicLabel.style.padding = "5px 0";
+    dynamicLabel.style.marginRight = "170px";
+    dynamicLabel.innerText = `Top ${this.topX} of PlanetType: ${this.planetType} up Lvl: ${this.minLevel}`;
+
+    // label for my planets
+    const myPLanetsLabel = document.createElement("label");
+    myPLanetsLabel.innerText = `only my planets?`;
+    myPLanetsLabel.style.marginRight = "20px";
+
+    // Checkbox for my planets ?
+    let MyPlanets = document.createElement("input");
+    MyPlanets.type = "Checkbox";
+    MyPlanets.checked = true;
+    MyPlanets.id = "myPlanets";
+    this.MyPlanets = MyPlanets;
+    MyPlanets.onchange = (evt) => {
+      try {
+        dynamicLabel.innerText = `Top ${this.topX} of PlanetType: ${this.planetType} up Lvl: ${this.minLevel}`;
+        this.renderPlanets();
+      } catch (err) {
+        console.error("Unable to change checkbox", err);
+      }
+    };
+
+    // This is the fuction for button upgrade that show current unconfirmed tx
+    function updateTx() {
+      dynamicLabel.innerText =
+        "Moves :  " +
+        df.getUnconfirmedMoves().length +
+        "  Upgrades :  " +
+        df.getUnconfirmedUpgrades().length +
+        "  Wormhole :  " +
+        df.getUnconfirmedWormholeActivations().length;
+    }
+
+    // This is the fuction that updates the transactions - change client response
+    function updateTxStatus() {
+      df.contractsAPI.contractCaller.queue.invocationIntervalMs = 100;
```

Comment:
Its fine if you set it for yourself but we cant put it inside plugins
i am quoting phated here from another pr:
`These can't be inside this plugin. The settings configured by the DF team are recommended by the xDai team directly. Additionally, the invocationIntervalMs was adjusted to 200ms earlier this week and no one has really been complaining since that change. Please remove these before acceptance`
https://github.com/darkforest-eth/plugins/pull/153

Reply:
resolved